charts/timescaledb-single/README.md

```diff
@@ -31,21 +31,24 @@ When configured for Backups to S3:
 
 ## Installing
 
+The Helm chart expects a Kubernetes secret named `postgres-secrets` to already be present in the same namespace. Copy the example secrets file `postgres-secrets-example.yaml` to `postgres-secrets.yaml`. Double-check that the secrets file is excluded from being comitted to version control (Git should automatically ignore the file).
```

Comment:
For those people running multiple deployments in the same namespace, this may not be the best way